### PR TITLE
Clean 1.3 branch and links

### PR DESCRIPTION
**Pull request by kolchfa-aws**
Clean 1.3 branch and links

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by hdhalter — APPROVED** [review]

File: _monitoring-your-cluster/logs.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+   ```yml
+   logger.org.opensearch.index.reindex: debug
+   ```
+
+   Modifying `opensearch.yml` makes the most sense if you want to reuse your logging configuration across multiple clusters or debug startup issues with a single node.
+
+2. Modify `log4j2.properties`:
+
+   ```
```
Comment (cwillum) [inline _monitoring-your-cluster/logs.md:69]: Do you want to add the language `bash` to these examples, and others that follow? (I'm thinking the Vale program).

File: _monitoring-your-cluster/logs.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+   # Define a new logger with unique ID of reindex
+   logger.reindex.name = org.opensearch.index.reindex
+   # Set the log level for that ID
+   logger.reindex.level = debug
+   ```
+
+   This approach is extremely flexible, but requires familiarity with the [Log4j 2 property file syntax](https://logging.apache.org/log4j/2.x/manual/configuration.html#Properties). In general, the other options offer a simpler configuration experience.
```
Comment (cwillum) [inline _monitoring-your-cluster/logs.md:76]: No comma OK
"... extremely flexible but requires ..."

File: _monitoring-your-cluster/logs.md [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+   - `${sys:os.logs.base_path}` is the directory for logs (for example, `/var/log/opensearch/`).
+   - `${sys:os.logs.cluster_name}` is the name of the cluster.
+   - `[%node_name]` is the name of the node.
+
+
+## Slow logs
+
+OpenSearch has two *slow logs*, logs that help you identify performance issues: the search slow log and the indexing slow log.
+
+These logs rely on thresholds to define what qualifies as a "slow" search or indexing operation. For example, you might decide that a query is slow if it takes more than 15 seconds to complete. Unlike application logs, which you configure for modules, you configure slow logs for indexes. By default, both logs are disabled (all thresholds are set to `-1`):
```
Comment (cwillum) [inline _monitoring-your-cluster/logs.md:94]: Just to make it perfectly clear that a threshold is applied to the two types of logs mentioned in the previous sentence.

These logs rely on thresholds to define what qualifies as a "slow" search or a "slow" indexing operation.

File: _monitoring-your-cluster/logs.md [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+
+```
+node1 | [2019-10-24T19:48:51,012][WARN][i.i.s.index] [node1] [some-index/i86iF5kyTyy-PS8zrdDeAA] took[3.4ms], took_millis[3], type[_doc], id[1], routing[], source[{"title":"Your Name", "Director":"Makoto Shinkai"}]
+```
+
+Slow logs can consume considerable disk space if you set thresholds or levels too low. Consider enabling them temporarily for troubleshooting or performance tuning. To disable slow logs, return all thresholds to `-1`.
+
+## Task logs
+
+OpenSearch can log CPU time and memory utilization for the top N memory expensive search tasks when task resource consumers are enabled. By default, task resource consumers will log the top 10 search tasks at 60 second intervals. These values can be configured in `opensearch.yml`.
```
Comment (cwillum) [inline _monitoring-your-cluster/logs.md:175]: I read this as a compound "N-memory-expensive search tasks". Maybe a second opinion for hyphenation?

**Review by cwillum — APPROVED** [review]
Looks good.

File: _monitoring-your-cluster/logs.md (hunk 4 above)
Comment (kolchfa-aws) [reply _monitoring-your-cluster/logs.md:175]: I just copied this file over to 1.3. I'll address the comments in another PR. Thanks!

**Pull request merged**